WalletLibrary/WalletLibrary/VerifiedId/VerifiableCredential/OpenID4VCIVerifiedID.swift

```diff
@@ -80,10 +84,39 @@ struct OpenID4VCIVerifiedId: InternalVerifiedId
         try container.encode(issuerName, forKey: .issuerName)
     }
     
-    /// TODO: implement in next PR.
     public func getClaims() -> [VerifiedIdClaim]
     {
+        guard let claims = raw.content.vc?.credentialSubject else
+        {
+            return []
+        }
+        
         var verifiedIdClaims: [VerifiedIdClaim] = []
+        for (claimReference, claimValue) in claims
+        {
+            let verifiedIdClaim = createVerifiedIdClaim(claimReference: claimReference,
+                                                        claimValue: claimValue)
+            verifiedIdClaims.append(verifiedIdClaim)
+        }
+        
         return verifiedIdClaims
     }
+    
+    private func createVerifiedIdClaim(claimReference: String, claimValue: Any) -> VerifiedIdClaim
+    {
+        if let claimDefinitions = configuration.credential_definition?.credential_subject,
+           let claimDisplayDefinitions = claimDefinitions["vc.credentialSubject.\(claimReference)"],
+           let localizedDefinition = claimDisplayDefinitions.getPreferredLocalizedDisplayDefinition(),
+           let label = localizedDefinition.name
+        {
+            return VerifiedIdClaim(id: label,
```

Comment:
Isn't type needed to be bubbled up here to format the claim in UI?

Reply:
Updated.